block-producer/src/lib.rs

```diff
@@ -29,9 +29,6 @@ pub(crate) type SharedRwVec<T> = Arc<RwLock<Vec<T>>>;
 // CONSTANTS
 // =================================================================================================
 
-/// The name of the block producer component
-pub const COMPONENT: &str = "miden-block-producer";
```

Comment:
I'm against removing the component.

Reply:
I'd prefer to keep the component too. @polydez what's the reasoning for removing it?